src/stories/Form/index.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import Form from "./component";
+
```

Comment:
Looks like this will be updated later, so you can involve the props.

Reply:
Updated it, added props, switched the naming around